test/CacheTest.cpp

```diff
@@ -5,10 +5,14 @@
 #include <gtest/gtest.h>
 
 #include <string>
+#include <string_view>
 
 #include "../src/util/Cache.h"
+#include "util/DefaultValueSizeGetter.h"
```

Comment:
Also fix the format of the include directly above.

Reply:
Done, see [f765e2c](https://github.com/ad-freiburg/qlever/pull/1111/commits/f765e2c675cf0b0bca4361f63dfc77c7e9a631c3).